src/__tests__/Collapse.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Collapse from '../Collapse';
+
+describe('Collapse', () => {
+  it('should render children', () => {
```

Comment:
Test coverage went down. Could you setup `jasmine.clock` here since you're using `setTimeout`.

```js
  beforeEach(() => {
    jasmine.clock().install();
  });

  afterEach(() => {
    jasmine.clock().uninstall();
  });
```

After prop changes, I believe you'll need to then use, `jasmine.clock().tick(400);` to move the virtual clock forward and test for the changes.